bower.json

```diff
@@ -31,8 +31,10 @@
     "generated-docs"
   ],
   "dependencies": {
-    "purescript-halogen": "^4.0.0",
-    "purescript-halogen-renderless": "^0.0.3"
+    "purescript-halogen": "^5.0.0-rc.3",
+    "purescript-halogen-renderless": "^0.0.4",
+    "purescript-variant": "^6.0.0",
+    "purescript-fixed-points": "^5.1.0"
```

Comment:
@davezuch this was for my brief foray into removing the newtype from `Action`, inspired by https://discourse.purescript.org/t/exploring-mutually-recursive-variants/424